kolibri/plugins/coach/assets/src/app.js

```diff
@@ -40,10 +34,17 @@ class CoachToolsModule extends KolibriModule {
           },
         },
         {
+          name: PageNames.COACH_RECENT_PAGE_CHANNEL_SELECT,
+          path: '/:classID',
+          handler: (toRoute, fromRoute) => {
+            recentActions.showChannelSelect(store, toRoute.params);
+          },
+        },
+        {
           name: PageNames.COACH_RECENT_PAGE,
-          path: RECENT_URL_PATTERN,
+          path: '/:classID/:channelID',
           handler: (toRoute, fromRoute) => {
-            actions.showRecentPage(store, toRoute.params);
+            recentActions.showReports(store, toRoute.params);
```

Comment:
Pass in `classID` and `channelID` as separate parameters to `showReports` to make it more explicit